SPECS-EXTENDED/supermin/supermin.spec

```diff
@@ -71,14 +56,15 @@ BuildRequires: gnupg2
 
 # These are required only to run the tests.  We could patch out the
 # tests to not require these packages.
+%if %{with_check}
```

Comment:
Didn't we remove the check section below? It seems that we don't need these then.

Reply:
Will add back in check section- removal was a mistake.